README.md

```
@@ -57,6 +57,32 @@ Simple and efficient HTTP mock server with specification written in `yaml`, `edn
         }
 ```
 
+**WebSocket Support**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)